src/core/renderers/webgl/utils/StencilMaskStack.js

```diff
@@ -6,8 +6,25 @@
  */
 function StencilMaskStack()
 {
+	/**
```

Comment:
You don't have editorconfig installed do you...grr...

You should install it, it will enforce consistent whitespaces for us, otherwise you get stuff like this where there are tabs instead of spaces.